packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -0,0 +1,328 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+import Stripe from 'stripe';
+import Container from 'typedi';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+import { AppConfig, AuthFirestore } from '../lib/types';
+
+const pckg = require('../package.json');
+
+const MONTHLY_NOTICE_DAYS = 14;
+const YEARLY_NOTICE_DAYS = 30;
+const SECONDS_TO_MILLISECONDS = 1000;
+
+// Represents a mapping of user uids to IP addresses
+interface IpAddressMap {
+  [key: string]: string;
+}
+
+// Firestore subscriptions contain additional expanded information
+// on top of the base Stripe.Subscription type
+interface FirestoreSubscription extends Stripe.Subscription {
+  customer: string;
+  plan: Stripe.Plan;
+  price: Stripe.Price;
+}
+
+const parseDryRun = (dryRun: boolean | string) => {
+  return `${dryRun}`.toLowerCase() !== 'false';
+};
+
+const parseBatchSize = (batchSize: string | number) => {
+  return parseInt(batchSize.toString(), 10);
+};
+
+const parseOutputFile = (outputFile: string) => {
+  return outputFile;
+};
```

Comment:
You got me curious&hellip; what this do?

Reply:
Ah, I'm just guessing it's for consistency with the other `parse*()` helpers and echos the input. Sure.

Reply:
Consistency mostly, but also the type conversion convenience.